atom/browser/renderer_host/atom_render_message_filter.cc

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "atom/browser/renderer_host/atom_render_message_filter.h"
+
+#include <stdint.h>
+
+#include <memory>
+#include <string>
+
+#include "base/bind.h"
+#include "base/bind_helpers.h"
+#include "base/logging.h"
+#include "base/stl_util.h"
+#include "base/task/post_task.h"
+#include "chrome/browser/browser_process.h"
+#include "chrome/browser/chrome_notification_types.h"
+#include "chrome/browser/content_settings/cookie_settings_factory.h"
+#include "chrome/browser/content_settings/tab_specific_content_settings.h"
+#include "chrome/browser/metrics/chrome_metrics_service_accessor.h"
+#include "chrome/browser/predictors/loading_predictor.h"
+#include "chrome/browser/predictors/loading_predictor_factory.h"
+#include "chrome/browser/predictors/preconnect_manager.h"
+#include "chrome/browser/profiles/profile.h"
```

Comment:
Why are we pulling in chromium profile/profile manager?  Couldn't this be done without pulling this in?

Reply:
@krunt can you answer the question above? Can we remove those files? thanks

Reply:
profile manager is not needed. I will remove it.
profile class declaration is needed in several places in chrome/browser/*
I tried to minimize related changes of chrome/browser/* and their interfaces.
actually I am not creating Profile class instances, just cast content::BrowserContext to Profile
 (as done in Profile::FromBrowserContext()) and pass to methods where profile is needed (checking for sure that is will be safe)

Reply:
@jkleinsc kindly check the code again. Thank you